e2e/cypress/integration/api/iam/projects_api_scanjob_ingestion.spec.ts

```diff
@@ -0,0 +1,208 @@
+            body: {}
+          });
+        }
+      });
+    });
+
+    // Delete the created node
+    cy.request({
+      headers: {
+        projects: ['*'],
+        'api-token': Cypress.env('ADMIN_TOKEN')
+      },
+      method: 'POST',
+      url: '/api/v0/nodes/search',
+      body: {
+        order: 'DESC',
+        page: 1,
+        per_page: 100
+      }
+    }).then((resp: Cypress.ObjectLike) => {
+      resp.body.nodes.forEach((node: any) => {
+        if (node.name === nodeName ) {
+          cy.request({
+            headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
+            method: 'DELETE',
+            url: `/api/v0/nodes/id/${node.id}`,
+            body: {}
+          });
+        }
+      });
+    });
+  });
+
+  it('when a project has a rule that matches a node\'s environment' +
+  ' successfully associates that node with the project', () => {
+    cy.request({
+      headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
+      method: 'POST',
+      url: '/api/v0/secrets',
+      body: {
+        'name': nodeCredName,
+        'type': 'ssh',
+        'data': [
+          {'key': 'username', 'value': targetUser},
+          {'key': 'password', 'value': decoded}
+        ],
+        'tags': []
+      }
+    });
+
+    cy.request({
+        headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
+        method: 'POST',
+        url: '/api/v0/secrets/search',
+        body: '{}'
+    }).then((getSecretResponse: Cypress.ObjectLike) => {
+      const secretId = getSecretResponse.body.secrets[0].id;
+      cy.request({
+          headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
+          method: 'POST',
+          url: '/api/v0/nodes',
+          body: {
+            'name': nodeName,
+            'manager': 'automate',
+            'target_config': {
+              'backend': 'ssh',
+              'host': host,
+              'secrets': [
+                secretId
+              ],
+              'port': 22
+            },
+            'tags': [{ 'key': 'environment', 'value': environment }]
+          }
+      }).then((nodeCreateResponse: Cypress.ObjectLike) => {
+        const nodeID = nodeCreateResponse.body.id;
+        cy.request({
+          headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
+          method: 'POST',
+          url: '/api/v0/compliance/scanner/jobs',
+          body: {
+            'name': 'test job',
+            'tags': [],
+            'type': 'exec',
+            'nodes': [nodeID],
+            'profiles': [
+              'https://github.com/dev-sec/linux-baseline/archive/master.tar.gz',
+              'https://github.com/dev-sec/ssh-baseline/archive/master.tar.gz'
+            ],
+            'retries': 1,
+            'node_selectors': []
+          }
+        }).then((createJobResponse: Cypress.ObjectLike) => {
+            // wait for the report to be ingested
+            cy.waitForComplianceNode(nodeID, nodeStart, nodeEnd, 10);
```

Comment:
This is where I am seeing it fail. 
@vjeffrey 
Is this waiting for the scan job to run and return a report?